lib/maze/document_server.rb

```diff
@@ -6,7 +6,10 @@ module Maze
   # HTTP server for a given document root
   class DocumentServer
     class << self
+      # Start the document server.  This is intended to be called only once per test run.
+      # Use manual_start for finer grained control.
       def start
+        @manual_start = false
```

Comment:
This will override the `@manual_start = true` in `manual_start` won't it?

Reply:
Good spot - have moved this to the `manual_stop` method.